From 7f3142fc5c57a741c61c7df9747e7148e7fdd8f6 Mon Sep 17 00:00:00 2001
From: Dominik Riebeling <Dominik.Riebeling@gmail.com>
Date: Sun, 10 May 2020 10:32:31 +0200
Subject: rbutil: Fix compiling 32bit Rockbox Utility on 64bit Linux.

Pass the machine options for libs to the c++ compiler as well.

Change-Id: I56bfbbe4aca4b3a7b86bb0d5323a1f7d78d4f60c
---
 rbutilqt/Makefile.libs | 1 +
 rbutilqt/rbutilqt.pro  | 3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/rbutilqt/Makefile.libs b/rbutilqt/Makefile.libs
index 24fe85e269..6d5c8c5354 100644
--- a/rbutilqt/Makefile.libs
+++ b/rbutilqt/Makefile.libs
@@ -28,6 +28,7 @@ export SYS_SPEEX
 export TARGET_DIR
 export RBBASE_DIR
 export CFLAGS=$(EXTRALIB_CFLAGS)
+export CXXFLAGS=$(EXTRALIB_CXXFLAGS)
 export CC=$(EXTRALIBS_CC)
 export CXX=$(EXTRALIBS_CXX)
 export AR=$(EXTRALIBS_AR)
diff --git a/rbutilqt/rbutilqt.pro b/rbutilqt/rbutilqt.pro
index a845c3ca36..f51d3608b3 100644
--- a/rbutilqt/rbutilqt.pro
+++ b/rbutilqt/rbutilqt.pro
@@ -33,7 +33,8 @@ unix:!mac:!noccache {
 MACHINEFLAGS = $$find(QMAKE_CFLAGS, -m[63][42])
 EXTRALIBS_OPTS += EXTRALIBS_CC=\"$$QMAKE_CC\"
 EXTRALIBS_OPTS += EXTRALIBS_CXX=\"$$QMAKE_CXX\"
-EXTRALIBS_OPTS += EXTRALIB_CFLAGS=\"$$MACHINEFLAGS\" \
+EXTRALIBS_OPTS += EXTRALIB_CFLAGS=\"$$MACHINEFLAGS\"
+EXTRALIBS_OPTS += EXTRALIB_CXXFLAGS=\"$$MACHINEFLAGS\"
 
 MYBUILDDIR = $$OUT_PWD/build/
 MYLIBBUILDDIR = $$MYBUILDDIR/libs/
-- 
cgit v1.2.1

